webpack/assets/javascripts/react_app/common/Tour/Tour.js

```diff
@@ -0,0 +1,82 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import Tour from 'reactour';
+import { noop } from '../helpers';
+import TourButton from './TourButton';
+
+const FOREMAN_COLOR = '#00759A';
+
+export const wrapComponentWithTour = (WrappedComponent, steps, id) => {
+  const tour = class TourComponent extends Component {
+    constructor(props) {
+      super(props);
+      if (this.props.alreadySeen === undefined) {
+        this.props.registerTour(id);
```

Comment:
`registerTour` should have prop-type

Why `alreadySeen === undefined`? Do we need an extra prop, `registered`?